src/idl_parser.cpp

```diff
@@ -15,7 +15,11 @@
  */
 
 #include <algorithm>
+#ifndef FLATBUFFERS_PREFER_PRINTF
```

Comment:
Now that the code below is changed, do we still need iostream and this ifdef at all?

Reply:
Done in 426759d.